docs/design/Versions and Releases.md

```diff
@@ -1,100 +1,107 @@
+## Version Number Syntax for Zonemaster components
 
 Published version numbers will be on the common x.y.z triplet form. We will refer to the x part as the _major version_, 
 the y part as the _minor version_ and the z part as the _patch version_. 
 
-## Versioning principle
+### Versioning principle
 
 We will follow the ideas of [Semantic Versioning](http://semver.org/). This is how they state it (version 2.0.0):
 
-```
-Given a version number MAJOR.MINOR.PATCH, increment the:
-
-    MAJOR version when you make incompatible API changes,
-    MINOR version when you add functionality in a backwards-compatible manner, and
-    PATCH version when you make backwards-compatible bug fixes.
-```
+>Given a version number MAJOR.MINOR.PATCH, increment the:
+>
+>    MAJOR version when you make incompatible API changes,
+>    MINOR version when you add functionality in a backwards-compatible manner, and
+>    PATCH version when you make backwards-compatible bug fixes.
 
-## Major version
+### Major version
 
 Changes in this number indicate changes in the outward-facing interface, and it means that users of 
 the interface probably have to do changes to their code, or changes of the usage.
 
-## Minor Version
+### Minor Version
 
-Changes in the minor version number indicate that there are functional changes that might change outward-facing interface,
-but not in a breaking fashion. A user of the interface should inform themselves on what the changes are, but does not have
-to make any modifications to their own code, unless the user wants to utilize updates that require changes to the code.
+Changes in the minor version number indicate that there are functional changes that is fully backward compatible.
+If the user wants to utilize the new features and functions, calls to Zonemaster might have to be changed.
+Interface changes are permitted, as long as they
+are of such a nature that they do not affect current users. That is, the interface after a minor version upgrade should be
+a strict superset of the interface as it was before. Adding new result messages is explicitly permitted at this level, and
+external code using Zonemaster must always be prepared to handle messages it has not seen before.
 
-## Patch Version
+### Patch Version
 
 Changes in the patch version number indicate internal changes in the library, that need not be visible to an external user
-in any way other than the reported test results being more correct. Small interface changes are permitted, as long as they
-are of such a nature that they do not affect current users. That is, the interface after a patch version upgrade should be
-a strict superset of the interface as it was before. Adding new result messages is explicitly permitted at this level, and
-external code using Zonemaster must always be prepared to handle messages it has not seen before.
+in any way other than the reported test results being more correct or previous bug does not show up.
+
+### Publication of releases
```

Comment:
This should probably be it's own 2nd level heading.

Reply:
Solved.